Workbooks/UpdateCompliance/UpdateComplianceHub/Update Compliance Hub.workbook

```diff
@@ -0,0 +1,763 @@
+            "label": "subscription",
+            "type": 6,
+            "value": null,
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "includeAll": true,
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            }
+          },
+          {
+            "id": "01c6d66e-3f87-4e8a-8be6-0accbe4f4036",
+            "version": "KqlParameterItem/1.0",
+            "name": "mappedWorkspace",
+            "label": "workspace",
+            "type": 5,
+            "isRequired": true,
+            "query": "resources\r\n| where type == \"microsoft.operationalinsights/workspaces\"\r\n| where \"{mappedSubscription:subscriptionId}\" == subscriptionId",
+            "crossComponentResources": [
+              "{mappedSubscription}"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "timeContext": {
+              "durationMs": 86400000
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "2776722c-672e-4a35-9c4b-3d8e74c1083f",
+            "version": "KqlParameterItem/1.0",
+            "name": "_SnapshotTime",
+            "label": "Time range",
+            "type": 1,
+            "description": "Time indicating the latest snapshot",
+            "isRequired": true,
+            "query": "UCClient\r\n| where TimeGenerated >= ago(7d)\r\n| summarize max(TimeGenerated)\r\n",
+            "crossComponentResources": [
+              "{mappedWorkspace}"
+            ],
+            "isHiddenWhenLocked": true,
+            "defaultValue": "value::1",
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "86531822-76f2-4c50-b766-542f46c8c41a",
+            "version": "KqlParameterItem/1.0",
+            "name": "isOnboarded",
+            "type": 1,
+            "query": "UCClient\r\n| summarize Total = count()\r\n| extend Result = iff(Total == 0, \"false\", \"true\")\r\n| summarize by Result",
+            "crossComponentResources": [
+              "{mappedWorkspace}"
+            ],
+            "isHiddenWhenLocked": true,
+            "queryType": 0,
+            "resourceType": "microsoft.operationalinsights/workspaces"
+          },
+          {
+            "id": "4f746974-5b8c-4f59-9277-4240b8aaf1b5",
+            "version": "KqlParameterItem/1.0",
+            "name": "currentWorkbook",
+            "type": 1,
+            "isGlobal": true,
+            "value": "Overview",
+            "isHiddenWhenLocked": true
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "parameters - 2"
```

Comment:
can you go through this  file and give these items descriptive names? since these will be templates customers will mainly use